tests/unit_tests/test_wave_06.py

```diff
@@ -193,8 +207,10 @@ def test_swap_best_by_category_no_match_is_false():
         my_priority="Clothing",
         their_priority="Clothing"
     )
+    
+    assert not result
```

Comment:
Good start! It would also be a good idea to check that the items in the inventories did not change, like you do for the other test.